assets/training/model_management/src/azureml/model/mgmt/processors/transformers/factory.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Factory module to create task based convertor classes."""
+
+from abc import ABC, abstractmethod
+from azureml.model.mgmt.processors.transformers.config import (
+    SupportedASRModelFamily,
+    SupportedDiffusersTask,
+    SupportedNLPTasks,
+    SupportedTasks,
+    SupportedTextToImageModelFamily,
+    SupportedVisionTasks,
+)
+from .convertors import (
+    NLPMLflowConvertor,
+    VisionMLflowConvertor,
+    WhisperMLflowConvertor,
+    StableDiffusionMlflowConvertor,
+)
+
+
+def get_mlflow_convertor(model_dir, output_dir, temp_dir, translate_params):
```

Comment:
Nit for next PR: Consider using "converter" instead of "convertor"
See https://grammarhow.com/convertor-or-converter/?expand_article=1